### DISCUSSION TITLE
payment issue

### DISCUSSION THREAD

**Discussion by rasulsh** [community/community#32082] [Copilot Conversations]
Hi there.
My account experienced a problem 13 days ago after I paid for active copilot subscription.
In the billing department, I opened a ticket, but I did not receive a response. I then opened a follow-up ticket 5 days later, but I never heard back from them.
Is there a way to solve my problem?

**Comment by lukasz-lapczyk** [comment]
Having issue with paying for Copilot too. Ticket opened 3 days ago, still no response 😞
